web-jehlomat/src/Components/Navigation/TitleBar.tsx

```diff
@@ -5,18 +5,19 @@ import { IconButton } from '@mui/material';
 
 interface ITitleBar {
     icon?: any | undefined;
+    mobile?: boolean | undefined;
```

Comment:
imho, pokud už má `?` nepotřebuje `undefined`
```suggestion
    mobile?: boolean;
```

Reply:
tak `icon` a `onIconClick` jsou taky